### PR TITLE
Travis CI support

### PR DESCRIPTION
**Pull request by Leonidas-from-XIV**

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -1,2 +0,0 @@
-wagerlabs (Joel Reymont)		inspired and contributed some poll code
```
Comment (andersfugmann) [inline CONTRIBUTORS]: I would not remove contributors. They did contribute to the code.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+sudo: required
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+env:
+  global:
+    - PACKAGE=zmq
+    - TESTS=true
+  matrix:
+    - OCAML_VERSION=4.02
```
Comment (andersfugmann) [inline .travis.yml]: Can only compile with Ocaml >= 4.03.0

File: README.md [hunk 3]
```diff
@@ -1,30 +1,35 @@
...
-======
-
 OCaml bindings for ZMQ 4.0
+==========================
 
 Requirements
 ------------
-* [OCaml >= 3.12](http://caml.inria.fr/)
+
+* [OCaml >= 3.12](https://ocaml.org/)
```
Comment (andersfugmann) [inline README.md]: Ocaml >= 4.03.0

File: README.md [hunk 4]
```diff
@@ -1,30 +1,35 @@
...
 * [Findlib](http://projects.camlcity.org/projects/findlib.html)
 * [ocaml-uint](https://github.com/andrenth/ocaml-uint)
 * [ZMQ 4.0.*](http://www.zeromq.org/intro:get-the-software)
-* [oasis](http://oasis.forge.ocamlcore.org/)
```
Comment (andersfugmann) [inline README.md]: These dependencies are not needed. Just refer to opam,
If you want them there, please refer to the libries listed in zmq.opam.

File: CONTRIBUTORS (hunk 1 above)
Comment (Leonidas-from-XIV) [reply CONTRIBUTORS]: This file has not been maintained for years, so either the list is available through Git or some other means or it is maintained but have a file crediting some people but not other people is not helpful at all.

Comment (Leonidas-from-XIV) [reply CONTRIBUTORS]: I've moved these two to the `README.md`, I think it should be an acceptable compromise.

**Pull request merged**